src/frontends/pytorch/src/frontend.cpp

```diff
@@ -155,6 +157,7 @@ void FrontEnd::normalize(const std::shared_ptr<ov::Model>& model) const {
     manager.register_pass<ov::pass::ConvertConvertLike>();
     manager.register_pass<ov::frontend::pytorch::pass::AtenIndexToSelect>();
 
+    manager.register_pass<ov::pass::MarkDequantizationSubgraph>(element::TypeVector{element::u8});
```

Comment:
Why only u8? List all types that we can support in theory (u8, i8, i4, u4).

Reply:
added more precisions